rd2/tbd
# Measure updates

To generate a TBD JSON using the Run Tab:
- Go to the Measures Tab
- Add the `Thermal Bridging & Derating (TBD)` measure
- Check the box to `Write TBD.json` in the `Thermal Bridging & Derating (TBD)` measure
- Go to the Run Tab and press run
- A starting point `tbd.out.json` will be written to the `files` directory of your model's companion directory

To generate a TBD JSON in Apply Measure Now:
- Go to Apply Measure Now
- Select the `Thermal Bridging & Derating (TBD)` measure
- Check the box to `Write TBD.json` in the `Thermal Bridging & Derating (TBD)` measure
- Run the measure, accept results, and save
- A starting point `tbd.out.json` will be written to the `files` directory of your model's companion directory

To verify a modified TBD JSON:
- Go to https://www.jsonschemavalidator.net/
- Copy contents of `tbd.schema.json` into the schema tab
- Copy contents of your modified `tbd.json` into the input JSON tab
- Fix any validation errors

To load a modified TBD JSON:
- Close the OpenStudio Application
- Copy your modified `tbd.json` into the `files` directory of your model's companion directory
- Open your model in the OpenStudio Application
- Check the box to `Load TBD.json` in the `Thermal Bridging & Derating (TBD)` measure
